tests/acceptance/steps/fixture-loading.py

```diff
@@ -10,6 +10,12 @@ def check_decorator_fixtures(context):
     pass
 
 
+@fixtures('behave-second-fixture.json')
+@given(u'a step with a second fixture decorator')
+def check_decorator_fixtures2(context):
```

Comment:
Please align the function name somewhat with the fixtures file. I'd rather have "check_decorator_second_fixture" than something that can be easily overlooked or mistaken. (Thanks!)